TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -179,6 +182,7 @@ public int GetTestCount()
 				foreach (var comparator in comparators) {
                     Console.WriteLine(comparator.paths.Result + " checked with " + comparator.GetType().Name);
 					var unit = new TestUnit(comparator, debug);
+                    unit.IsCobolLanguage = IsCobolLanguage;
```

Comment:
strange indent

Reply:
I don't see it

Reply:
The file has a mix of tabs and spaces so indent is inconsistent. But it's ok you can leave it as it is.